app/api/auth.py

```diff
@@ -39,12 +39,20 @@ def basic_auth_error():
 #     return user.check_password(password)
 
 
+# @token_auth.verify_token
+# def verify_token(token):
+#     g.current_user = User.check_token(token) if token else None
+#     print(g.current_user)
+#     return g.current_user is not None
+
 @token_auth.verify_token
 def verify_token(token):
-    g.current_user = User.check_token(token) if token else None
-    print(g.current_user)
-    return g.current_user is not None
-
+   print("why")
```

Comment:
remove the printing in this file (if you need to print, you should be using a logger)

Reply:
LOL, I forgot that,
actually, as I said I need to clean up the code a lot,

prints like these are just me know which line is printing